src/app/screens/MakeInvoice.tsx

```diff
@@ -83,64 +86,68 @@ function MakeInvoice({
   }
 
   return (
-    <div>
-      <PublisherCard title={origin.name} image={origin.icon} />
-      <div className="py-4">
-        <Container maxWidth="sm">
-          <div className="mb-8">
-            <div className="mb-4">
-              <p className="font-semibold text-gray-500 mb-4">
-                {origin.host} requests an invoice:
-              </p>
-              <div className="mt-4">
-                {amountEditable && (
-                  <>
-                    <TextField
-                      id="amount"
-                      label="Amount"
-                      type="number"
-                      min={invoiceAttributes.minimumAmount}
-                      max={invoiceAttributes.maximumAmount}
-                      value={value}
-                      onChange={(e) => handleValueChange(e.target.value)}
-                    />
-                    <SatButtons onClick={handleValueChange} />
-                  </>
-                )}
-                {!amountEditable && (
-                  <dl className="dark:bg-surface-02dp pt-4 overflow-hidden">
-                    <Dt>Amount</Dt>
-                    <Dd>{invoiceAttributes.amount}</Dd>
-                  </dl>
-                )}
-                {error && <p className="mt-1 text-red-500">{error}</p>}
-              </div>
-              <div className="mt-4">
-                {memoEditable && (
+    <div className="h-full flex flex-col overflow-y-auto no-scrollbar">
+      <div className="text-center text-xl font-semibold dark:text-white py-2 border-b border-gray-200 dark:border-neutral-500">
```

Comment:
should this be a header component? 
we have some duplication there in div and css classes.